diff --git a/node_modules/@langchain/community/dist/tools/wikipedia_query_run.js b/node_modules/@langchain/community/dist/tools/wikipedia_query_run.js
index 6a3bcd2..fc0a8e1 100644
--- a/node_modules/@langchain/community/dist/tools/wikipedia_query_run.js
+++ b/node_modules/@langchain/community/dist/tools/wikipedia_query_run.js
@@ -156,6 +156,7 @@ export class WikipediaQueryRun extends Tool {
         const searchParams = new URLSearchParams({
             action: "query",
             list: "search",
+            origin: "*",
             srsearch: query,
             format: "json",
         });
@@ -172,6 +173,7 @@ export class WikipediaQueryRun extends Tool {
             explaintext: "true",
             redirects: redirect ? "1" : "0",
             format: "json",
+            origin: "*",
             titles: page,
         });
         const response = await fetch(`${this.baseUrl}?${params.toString()}`);
